docs/docs/development/hardware-integration/new-shield.mdx

```diff
@@ -60,13 +64,6 @@ Neither of these should be moved out of their parent directory. The other files
 
 ## New Shield Directory
 
-:::note
-This guide describes how to add a shield to an independently managed Zephyr module repository. This is the
-preferred way to handle boards and shields moving forward in ZMK, although the tooling to make this easier
-for users is still improving. ZMK does have a collection of boards/shields in the ZMK main repository, which are planned to be phased out, but until that is complete, there _may_ be a few select scenarios where adding your keyboard to ZMK itself is preferred. Due the volume of PRs and the focus of ZMK development not being merging of keyboard PRs, you are highly encouraged to use an out-of-tree Zephyr module repository to manage your definitions. Should you choose to try to get your keyboard included in ZMK main repository, the paths in the rest of the guide would be nested under the `app/` folder there instead. For example, `boards/shields/<keyboard_name>` should now be
-`app/boards/shields/<keyboard_name>`.
```

Comment:
Do you think it'd be better to leave the descriptive portion? e.g. it could simply read:

> This guide describes how to add a shield to an independently managed Zephyr module repository. This is the preferred way to handle boards and shields moving forward in ZMK, although the tooling to make this easier for users is still improving. ZMK does have a collection of boards/shields in the ZMK main repository, which are planned to be phased out. Due the volume of PRs and the focus of ZMK development not being merging of keyboard PRs, you are highly encouraged to use an out-of-tree Zephyr module repository to manage your definitions.

Reply:
We could include it. I'm not really sure it's that helpful for the reader though - it's a lot of text to essentially explain "add your shield the way this guide tells you to, using modules".

Reply:
Sure, it only clears up questions about "what about all these existing keyboards." I'm fine to remove it.